packages/components/src/components/positions-drawer-card/positions-drawer-card.jsx

```diff
@@ -47,22 +48,23 @@ const PositionsDrawerCard = ({
 }) => {
     const is_accumulator = isAccumulatorContract(contract_info.contract_type);
     const is_multiplier = isMultiplierContract(contract_info.contract_type);
+    const is_turbos = isTurbosContract(contract_info.contract_type);
     const is_vanilla = isVanillaContract(contract_info.contract_type);
     const is_crypto = isCryptoContract(contract_info.underlying);
     const has_progress_slider = !is_multiplier || (is_crypto && is_multiplier);
     const has_ended = !!getEndTime(contract_info);
+    const has_regular_style = !is_accumulator && !is_multiplier && !is_turbos;
     const is_mobile = isMobile();
     const contract_card_classname = classNames('dc-contract-card', {
-        'dc-contract-card--green': !is_accumulator && !is_multiplier && profit_loss > 0 && !result,
-        'dc-contract-card--red': !is_accumulator && !is_multiplier && profit_loss < 0 && !result,
+        'dc-contract-card--green': has_regular_style && profit_loss > 0 && !result,
```

Comment:
should this be `profit_loss > 0 && !results` ?  and get rid `has_regular_style` all together? And have designer update figma design all together.

Reply:
@henry-deriv we would anyway resolve the conflict when https://github.com/binary-com/deriv-app/pull/8883 is merged. but it's a better idea to apply it in this branch too before that.

Reply:
@kate-deriv could you please apply this to our branch as the author of https://github.com/binary-com/deriv-app/pull/8883? or do you think it's better if we wait for your card's release for the reason that the requirement is implemented in your card, not in our PR dedicated to adding Turbos trade type on top of what we have in production? so it might simply fall out of this PR's scope..